nautiluszim/scraper.py

```diff
@@ -145,14 +145,18 @@ def run(self):
         self.check_branding_values()
 
         # download archive
-        self.download_archive()
+        if self.archive:
+            self.download_archive()
 
         if not self.collection:
             self.collection = self.extract_to_fs("collection.json")
             if not self.about:
                 self.extract_to_fs("about.html", failsafe=True)
 
-        self.test_collection()
+        if self.collection and not self.archive:
```

Comment:
At this point `self.collection` always exists (would have failed earlier if not)